vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -386,11 +376,28 @@ bool ChakraRuntime::isHostFunction(const facebook::jsi::Function &obj) const {
 }
 
 facebook::jsi::Array ChakraRuntime::getPropertyNames(const facebook::jsi::Object &object) {
```

Comment:
>getPropertyNames [](start = 36, length = 16)

Since we are temporarily reverting back to the implementation that (incorrectly) returns non-enumerable properties, we fail JsiRuntimeUnitTests.ObjectTest at line 158 in JsiRuntimeUnitTests.cpp. We should temporarily disable this test as well.

Reply:
Hum It looks like the JSI unit tests are no longer being executed in our CI loop. I'll follow up on that.